app/views/shelters/index.html.erb

```diff
@@ -1,11 +1,18 @@
 <h1>All Shelters</h1>
+
+<%= link_to "Sort by Number of Adoptable Pets", "/shelters?order_by_num_adopatable=true"  %>
+<%= link_to "Sort by Shelter Name", "/shelters?order_by_name=true"  %>
+
+<section class="shelter_list">
```

Comment:
I used the class indicator for this section so that I was able to test for the order of the shelters.